lib/all-rules.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
can you link this to lib/index.js -> base -> rules?
I see no reason to have this in multiple places.